map2loop/map2model_wrapper.py

```diff
@@ -133,9 +133,9 @@ def run(self, verbose_level: VerboseLevel = None):
         }
         # TODO: Simplify. Note: this is external so have to match fix to map2model module
         run_log = map2model.run(
-            os.path.join(self.map_data.tmp_path, "map2model_data"),
-            os.path.join(self.map_data.tmp_path, "map2model_data", "geology_wkt.csv"),
-            os.path.join(self.map_data.tmp_path, "map2model_data", "faults_wkt.csv"),
+            os.path.join(self.map_data.map2model_tmp_path),
```

Comment:
Can you used pathlib.Path(self.map_data.map2model_tmp_path)/"geology_wkt.csv"... could also change map_data.map2model_tmp_path to a pathlib object?

Reply:
https://github.com/Loop3D/map2loop/pull/112/commits/fb5efaefb1709b5e006c61820c3582342c74908b
https://github.com/Loop3D/map2loop/pull/112/commits/8892241829be9c59230d2d213c94dafa93ecdbfd